gulp-tasks/workbox/workbox-reference-docs-v3.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Add the JSDoc description to this file and include at least `@fileoverview` and `@author`

Reply:
Done.